MultimodalQnA/docker_image_build/build.yaml

```diff
@@ -59,7 +59,7 @@ services:
       dockerfile: comps/dataprep/multimodal/redis/langchain/Dockerfile
     extends: multimodalqna
     image: ${REGISTRY:-opea}/dataprep-multimodal-redis:${TAG:-latest}
-  whisper-service:
+  whisper:
```

Comment:
I'd make sure there are no references in the repo or `GenAIExamples` for this service name if we chage it.

Reply:
That's what I did and I saw the references in the `compose.yaml` files, but @dmsuehir thinks those don't have to be changed.